### ISSUE TITLE
CustomEncodeProfiles

### ISSUE THREAD

**Issue by hanami117** [dfaker/WebmGenerator#117]
![image](https://github.com/user-attachments/assets/2d837104-be06-424d-9a86-3f5ce42f24b1)
I dont know if its me or I cant seem to understand how customencodeprofiles work. In my Picture I have those options, but in my customEncodeProfiles there are many more put there by default. How can I use them or see them?

**Comment by hanami117** [comment]
![image](https://github.com/user-attachments/assets/4023c43f-0e2f-416b-94c1-a1567c2e9768)
For Context. I am on Linux too.

**Comment by Mojavve** [comment]
Are you using a fresh repo copy of the program or the old release version (3.41.1)?
If you're using the old release version it could be this issue #112 

Solutions being:

- Download the repo instead and just use that (the version on the releases page is really out of  date).

**OR** 

- Rename your folder to have the same casing as the old code (Probably something like `customEncodeProfiles`).

**Comment by hanami117** [comment]
> Are you using a fresh repo copy of the program or the old release version (3.41.1)? If you're using the old release version it could be this issue #112
> 
> Solutions being:
> 
>     * Download the repo instead and just use that (the version on the releases page is really out of  date).
> 
> 
> **OR**
> 
>     * Rename your folder to have the same casing as the old code (Probably something like `customEncodeProfiles`).

First of all thanks for the reply.
I just reinstalled my machine and did a fresh pull from the repo (DOWNLOAD ZIP). I had to remember the folder from customencodeProfiles to customEncodeProfiles. Missed a capital E, but the issue isnt solved. I still dont see the presets.
Also, in this new install drag and drop of files works and doesnt work. Dont know based on what.
Plus, when I drop a file in, instead of playing inside the app it opens mpv outside.
![image](https://github.com/user-attachments/assets/65ffba8b-cc04-4599-832b-7a6467f27228)

**Comment by Mojavve** [comment]
On this new install, did you try opening WmG and checking the profiles before renaming the folder?  On the newest commit, renaming the folder would break it since it was fixed for the default folder name.

I see that you're using Konsole so I assume you're on KDE.  Are you logging into Plasma (Wayland) or Plasma (X11)?
It's just a shot in the dark but if you are on Wayland, do these issues persist on an X11 session?

Are you certain you have python-mpv installed?
What distro are you on?

I use AwesomeWM which is an X11 window manager and I have none of these issues but considering the nature of Linux it's hard to understand more without information on your setup since installs can be so wildly different from one another.

**Comment by hanami117** [comment]
Hello, I am on Wayland on Arch Linux.

I installed python-mpv with pip, but no luck
EDIT: which was installed with requisites.txt anyway

Switching to X11 solves the MPV opening outside, but I cannot drag and drop files inside the software at all.

Issue that I mentioned here: but wasnt this consistent before
https://github.com/dfaker/WebmGenerator/issues/115

Other than that, the custom profiles still do not appear.

> ry opening WmG and checking the profiles before renaming the folder? On the newest commit, renaming the folder would break it since it was fixed for the default folder name.

I named it back, but still doesnt work unfortunately.
